query/querytest/compiler.go

```diff
@@ -0,0 +1,20 @@
+package querytest
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/querytest"
+	"github.com/influxdata/flux/stdlib/influxdata/influxdb/v1"
+	"github.com/influxdata/influxdb/query/stdlib/influxdata/influxdb"
+)
+
+// FromInfluxJSONCompiler returns a compiler that replaces all From operations with FromJSON
```

Comment:
Godoc should be a complete sentence, with punctuation at the end. https://blog.golang.org/godoc-documenting-go-code